src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketMirroredRepository.java

```diff
@@ -0,0 +1,54 @@
+package com.atlassian.bitbucket.jenkins.internal.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BitbucketMirroredRepository {
+
+    private final boolean available;
+    private final Map<String, List<BitbucketNamedLink>> links;
+    private final String mirrorName;
+    private final int repositoryId;
+    private final String status;
```

Comment:
what are the values for `status`? I looked but didn't see any json files in the repository, but it feels like it should be an enum rather than a `String`?

Reply:
I should have looked further, its a enum and I will update it to be consistent with Bitbucket